GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -110,7 +110,10 @@ public string RunServiceVerb(string argument)
 
         public string ReadConfig(string key)
         {
-            return this.CallGVFS($"config {key}", failOnError: true).TrimEnd('\r', '\n');
+            // failOnError is set to false because gvfs config read can exit with
+            // GenericError when the key-value is not available in config file. That
+            // is normal.
+            return this.CallGVFS($"config {key}", failOnError: false).TrimEnd('\r', '\n');

```

Comment:
I recommend passing `bool failOnError` in as a parameter like we do for `Prefetch`.  That way the caller is forced to declare whether or not a failure is allowed/expected (it's generally safe to assume that `failOnError: true` is okay, but I don't think we should do that for `failOnError: false`).